pkg/operator/controller/ingress/dns.go

```diff
@@ -105,6 +112,7 @@ func desiredWildcardDNSRecord(ic *operatorv1.IngressController, service *corev1.
 		recordType = iov1.ARecordType
 		target = ingress.IP
 	}
+	clusterTag := fmt.Sprintf("kubernetes.io_cluster.%s", infraID)
```

Comment:
Where is this label documented?

Reply:
@patrickdillon correct me if I am wrong. I can't find the documentation for this but we started using this tag only for azurestack hoping to destroy records based on tags for all platforms if they'd allow it. I was hoping this would set the metadata of the A record on Azure but figured out that's not how it works so changed the logic a little.

Reply:
I believe this is [an upstream k8s convention for aws](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/legacy-cloud-providers/aws/tags.go#L48-L51) which the installer has adopted for all platforms. Wherever possible, cluster resources should be labeled as `owned` to denote they are managed by the cluster & therefore safe to delete. 

That is the idea, here, we would like to label this DNS record so we can find it (DNS records are created outside of azure resource groups) and know it is safe to delete.